src/app/tickets/ticket-list/ticket-list.component.ts

```diff
@@ -10,18 +18,75 @@ import { TicketService } from '../ticket.service';
 })
 export class TicketListComponent implements OnInit {
   tickets: Ticket[];
+  ticketsTable = new MatTableDataSource(this.tickets);
+  displayedColumns: string[] = [
+    'ticketNo',
+    'sdType',
+    'kTool',
+    'name',
+    'description',
+    'icons',
+  ];
+
+  private ticketChangeSub: Subscription;
+
+  @ViewChild(MatSort, { static: true }) sort: MatSort;
+  @ViewChild(MatPaginator, { static: true }) paginator: MatPaginator;
 
   constructor(
     private ticketService: TicketService,
-    private router: Router,
-    private route: ActivatedRoute
+    private dialog: MatDialog
   ) {}
 
-  ngOnInit() {
+  ngOnInit(): void {
     this.tickets = this.ticketService.getTickets();
+    this.ticketsTable = new MatTableDataSource<Ticket>(this.tickets);
```

Comment:
What is the ticketsTable doing? I don't see it referenced anywhere.

Reply:
it was my failed attempt to get the filter to work... I'll remove it and figure it out in the next user story.